doc/nitrocli.1

```diff
@@ -318,6 +318,37 @@ The admin PIN cannot be unblocked.
 This operation is equivalent to the unblock PIN option provided by \fBgpg\fR(1)
 (using the \fB\-\-change\-pin\fR option).
 
+.SS Extensions
+In addition to the above built-in commands, \fBnitrocli\fR supports
+user-provided functionality in the form of extensions. An extension can be any
+executable file whose filename starts with "nitrocli-" and that is discoverable
+through lookup via the \fBPATH\fR environment variable. Those executables can be
+invoked as regular subcommands (without the need of the prefix; e.g., an
+extension with the name "nitrocli-otp-cache" could be invoked as "nitrocli
+otp-cache").
+
+\fBnitrocli\fR conveys basic configuration information to any extension being
+started this way. Specifically, it may provide all environment variables as
```

Comment:
Isn’t “may” too weak here?  It *will* provide the environment variables if the options are set when calling `nitrocli` (via arguments, environment variables or configuration).

Reply:
I had changed that to "may" because not all the variables are set all the time. If no model is specified (either via `NITROCLI_MODEL`, `--model`, or config), it won't be set by `nitrocli` either (same for USB path). I felt that "will" would be even more misleading. So it's exactly about that "if" part you mention.

I changed it to:
```
\fBnitrocli\fR conveys basic configuration information to any extension being
started this way. Specifically, it will set each environment variable as
described in the Configuration subsection of the Environment section, if
the corresponding \fBnitrocli\fR program configuration was set. In addition, the
following variable will be set unconditionally:
```
Let me know if you have a better suggestion.